Tests/VaporTests/URLEncodedFormTests.swift

```diff
@@ -220,6 +220,21 @@ final class URLEncodedFormTests: XCTestCase {
         XCTAssertEqual(decodedCustom, toEncode)
     }
 
+    func testGH2518() throws {
```

Comment:
Small nit, can we rename this so it's clearer what the test achieves
```suggestion
    func testOptionalDateEncodingAndDecoding_GH2518() throws {
```